### DISCUSSION TITLE
Cant find var/sqlite/dex.db

### DISCUSSION THREAD

**Discussion by AnkitaBarbi** [dexidp/dex#1981] [Q&A] [answered]
Trying to build up dex via Dockerfiles, using the original config.yaml for storage (sqlite3), but my docker is failing as it cant find var/sqlite/dex.db. Neither its present in repo. What am I missing here?

**Comment by nabokihms** [comment, marked as answer]
Hello! 
If sqlite3 database file doesn't exist, dex will create it on startup. However, /var/sqlite folder is missing in the container.

I think you should change the path to  something like /var/dex/dex.db to make it works inside of the container.

**Reply by AnkitaBarbi** [reply]
Sure, let me give it a try, and will update here on how it goes

**Reply by AnkitaBarbi** [reply]
Thanks it works
